### DISCUSSION TITLE
Whitelabel Error Page

### DISCUSSION THREAD

**Discussion by hkhattabii** [graphql-java/graphql-java#2187] [Q&A]
Hi all,

I am trying to run the tutorial project provided from the official link to use it for my Spring boot project but I cannot access to the GraphQL playground.

I have cloned the tutorial repo and launched it without making any modification but I have this error :

Whitelabel Error Page
This application has no explicit mapping for /error, so you are seeing this as a fallback.

Sat Feb 06 12:05:24 CET 2021
There was an unexpected error (type=Bad Request, status=400).
Required String parameter 'query' is not present

Someone can help me please ?

Thank you in advance :)

HK

**Comment by bbakerman** [comment]
> Required String parameter 'query' is not present

Sounds like there will be  a stack trace showing the erros in your log. Can you post that

**Reply by hkhattabii** [reply]
I resolved the problem, It seems that graphiql it nos included in the library. I am able to do query from graphql playground

**Reply by bdeckard92** [reply]
How did you resolve this? I get the same error
